.ci/generate_bear_requirements.py

```diff
@@ -163,6 +164,20 @@ def get_cabal_requirements(requirements):
     return _get_requirements(requirements, '==')
 
 
+def get_languages(bears):
+    language_dict={}
+<<<<<<< HEAD
```

Comment:
You need to solve this merge conflict

Reply:
Sould i squash all commits

Reply:
First of all, try to solve the conflict and then squash it. (In case you're not able to find where to solve the issue, try to squash them all, it will raise a merge conflict error which you can solve from your editor)